RecoLocalCalo/EcalRecProducers/python/ecalMultiFitUncalibRecHit_cfi.py

```diff
@@ -14,7 +14,8 @@
       activeBXs = cms.vint32(-5,-4,-3,-2,-1,0,1,2,3,4),
       ampErrorCalculation = cms.bool(True),
       useLumiInfoRunHeader = cms.bool(True),
-  
+
+      gainSwitchEBMaxSample = cms.bool(False),
```

Comment:
in 90X I have used a notation similar to the other customization methods. 
For consistency I will change the notation of 80X branch to be the same as in 90X.